scripts/bash/downloadEnvironmentVariables.sh

```diff
@@ -72,7 +72,7 @@ load_env_file_from_bw() {
     echo -e "${GREEN}✅ Downloaded variables for ${BOLD}${FILE_NAME}${RESET} → $ENV_FILE_PATH"
 }
 
-for PACKAGE in "${PACKAGES[@]}"; do
+for PACKAGE in $PACKAGES; do
```

Comment:
@jaskfla this fixes the issue by reverting to the previous syntax, but happy if you have a better change :)